i18n/bn-IN.properties

```diff
@@ -849,7 +849,7 @@ be.versionCollapsed = {name} {actionMessage} v{versions}
 # Message displayed in the activity feed for a deleted version. {name} is the user who performed the action. {version_number} is the file version string.
 be.versionDeleted = {name} v{version_number}মুছে দিয়েছে
 # Message displayed in the activity feed to represent the range of versions actioned by multiple users. {numberOfCollaborators} is a number. {actionMessage} is the action. {versions} is a range of versions.
-be.versionMultipleUsersCollapsed = {numberOfCollaborators} collaborators {actionMessage} v{versions}
+be.versionMultipleUsersCollapsed = {numberOfCollaborators} কোলাবোরেটর {actionMessage} v{versions}
```

Comment:
is it trying to handle it's own translations?

Reply:
Certainly looks like it.

Reply:
Do we need to address that or is the PR good to go?